node/engine/parse/types.go

```diff
@@ -0,0 +1,60 @@
+package parse
+
+import (
+	antlr "github.com/antlr4-go/antlr/v4"
+)
+
+// Position is a Position in the parse tree. It represents a range of line and column
+// values in Kuneiform source code.
+type Position struct {
+	// Set is true if the position of the Position has been set.
+	// This is useful for testing parsers.
+	IsSet     bool `json:"-"`
```

Comment:
No real complaint about the tag, but it could be unexported, with an `IsSet()` method if needed from a different package.

Reply:
Ah we actually dont need JSON tags here anymore. Previously, this was used by the IDE, however now that all parsing is post-consensus, there is no need.